cmd/sachet/config.go

```diff
@@ -53,7 +53,7 @@ var config struct {
 		Exotel       exotel.ExotelConfig
 		CM           cm.CMConfig
 		MailruIM     mailruim.MailruIMConfig
-		Telegram     telegram.TelegramConfig
```

Comment:
nice. This is wrong for sometime. We should change the struct names for the others too one day 😅

Reply:
Yep, I can do this later in another PR 👍